src/components/common/EntityInfo.vue

```diff
@@ -26,6 +26,7 @@
               :key="getBusinessId"
               :axios="axios"
               :businessId="getBusinessId"
+              :url="commentsUrl"
```

Comment:
To override the default URL -- see https://github.com/bcgov/bcrs-shared-components/blob/a963fdfc3710f0a98ab0d58034e887eeaf8dacaa/src/components/staff-comments/StaffComments.vue#L182.